subprojects/version-control/src/test/groovy/org/gradle/vcs/git/internal/GitVersionControlSystemSpec.groovy

```diff
@@ -122,7 +131,7 @@ class GitVersionControlSystemSpec extends Specification {
     def "error if working dir repo is missing the remote"() {
         given:
         def target = tmpDir.file("workingDir")
-        GitVersionControlSpec spec = new GitVersionControlSpec()
+        GitVersionControlSpec spec = new DefaultGitVersionControlSpec()
         spec.url = repo.url
```

Comment:
:o: nit, it might be cleaner to move the spec setup into the `setup()` method too.

Reply:
Done